### PR TITLE
New postprocessing modules and name changes in metrics

### PR DESCRIPTION
**Pull request by colehurwitz31**

### CODE REVIEW

**Comment by colehurwitz31** [comment]
Added new postprocessing curation sorting extractors which could be useful

**Comment by colehurwitz31** [comment]
Added option for n_jobs in sorting comparison in case people want to choose number of cores

**Comment by alejoe91** [comment]
@colehurwitz31 Iremoved the extralines in the docsrtings! We should land on a style here eheheh

File: spiketoolkit/postprocessing/threshold_max_ISI.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        isi_ratio_list = compute_ISI_violation_ratio(self._sorting, self._sampling_frequency)
+        for i, unit_id in enumerate(self._sorting.get_unit_ids()):
+            if isi_ratio_list[i] >= self._max_ISI_threshold:
+                units_to_be_excluded.append(unit_id)
+        self.exclude_units(units_to_be_excluded)
+
+
+def threshold_max_ISI(sorting, sampling_frequency, max_ISI_threshold=0.4):
+    '''
+    Excludes units with SNR greater than or equal to the max_SNR_threshold.
```
[the remark this answers is not in the document]
Comment (colehurwitz31) [reply spiketoolkit/postprocessing/threshold_max_ISI.py]: Fixed

File: spiketoolkit/postprocessing/postprocessinglist.py [hunk 2]
```diff
@@ -1,8 +1,12 @@
 from .threshold_min_num_spikes import threshold_min_num_spikes, ThresholdMinNumSpike
+from .threshold_min_SNR import threshold_min_SNR, ThresholdMinSNR
```
[the remark this answers is not in the document]
Comment (colehurwitz31) [reply spiketoolkit/postprocessing/postprocessinglist.py]: Added

File: spiketoolkit/validation/__init__.py [hunk 3]
```diff
@@ -1,2 +1,2 @@
-from .biophysicalmetrics import get_ISI_violation_ratio
+from .biophysicalmetrics import compute_ISI_violation_ratio
```
[the remark this answers is not in the document]
Comment (colehurwitz31) [reply spiketoolkit/validation/__init__.py]: Updated

**Pull request merged**